app/views/posts/index.html.erb

```diff
@@ -1,3 +1,40 @@
-<div class="index-allPosts-page">
-    <h2>All User Posts</h2>
-</div>
\ No newline at end of file
+<div class="index-page">
+  <div class="user-tile card">
+    <img class="user-image" src=<%= @user.photo %> />
+    <div class="user-info">
+      <h2 class="user-name"><%= @user.name %></h2>
+      <p>
+        Number of <%= @user.posts_counter == 1 ? 'Post' : 'Posts' %>
+        <%= @user.posts_counter %>
+      </p>
+    </div>
+  </div>
+
+  <ul class="user-posts-display">
+    <% @posts.each do |post| %>
+      <li>
+      <div class='post-list'>
+          <%= link_to post.title, user_post_path(user_id: @user.id, id:post.id) %>
+        <p>
+          <%= sanitize post.text[0..200].gsub("\n", '<br />') %>...
+        </p>
+        <div class="post-interactions">
+          Comments: <%= post.comments_counter %>,
+          Likes: <%= post.likes_counter %>
+        </div>
+        </div>
+        <ul class="post-comments">
+          <% post.most_recent_comments.each do |comment| %>
+            <li>
+              <h4><%= comment.user.name %>:</h4>
+              <p><%= comment.text[0..100]... %></p>
+            </li>
+          <% end %>
+        </ul>
+        <%= 'No comments yet!' if post.comments.empty? %>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
- [ ] As mentioned by the previous reviewer, as per the project requirements, on the Posts Index page, if a post has any comments, it should utilize the most_recent_comments method in the posts model to fetch the 5 most_recent comments. However, in your case, this method does not work as expected, instead, it displays all the comments belonging to a post that I created (6 . You might need to look into the post model and check where the issue is. Kindly fix this issue.

### Comments error.

![comment-error](https://github.com/Yacoubou-seidou/RailsBlog/assets/26597432/42f984e2-b225-44f0-8dae-06b0536be5c1)

Reply:
@Qoosim Sorry but I fixed that you are not on the right page this page is the `/posts/:id` so all comments are there check the preview reviewer request is the path `/posts`. I can't find your name in the form dropdown, Kindly give me your full name so I can complete the form to ask for a second opinion thanks.
![blogg](https://github.com/Yacoubou-seidou/RailsBlog/assets/84572891/0aa38e21-1ef2-4dc5-a716-21357ab67d7a)